app.py

```diff
@@ -28,7 +28,24 @@ def update():
 @app.route("/thank-you")
 def success_page():
     return render_template("thank-you.html")
-
+# This is for registerAlt.html, which is my slow progression on using Bootstrap to make the Register Page mobile friendly #
+@app.route("/registerV2", methods=('GET','POST'))
+def improved_register():
+    return render_template("registerAlt.html")
+###########################################################################################################################
+@app.route("/register", methods=('GET','POST'))
+def register_page():
+    print("Can You See This")
+    if request.method == 'POST':
+        firstName = request.form['firstName']
+        lastName = request.form['lastName']
+        email = request.form['email']
+        major = request.form['major']
+        year = request.form['year']
+        username = firstName + lastName
```

Comment:
@guilleeh @MantaRey @jaime-leal  we might want to pass these variables into a pyrebase function. we can write the function ourselves

Reply:
Yes! Definitely we will need to do that. I was just seeing if I was able to read from the form and temporarily "store" the data.